proposals/distribution.md

```diff
@@ -103,11 +103,11 @@ The following entries should be added to the [scope table][scope]:
 * “Specifying a distribution method”.
     This entry replaces part of the previous “Creating Reference spec for optional DNS based naming & distribution” and “Standardizing on a particular Distribution method” entries.
 
-    Retrieving image indexes covers the current “tag listing” (e.g. “what named manifests are in `library/busybox`?”), because tags are entries in the image format's [`manifests` array][manifests].
-    Other tag-listing endpoints needed for backwards-compatibility are therefore in scope as well.
+    Tag-listing (e.g. “what named manifests are in `library/busybox`?”) endpoints are in scope and required for backwards compatibility with clients.
+    External methods for listing tags remain compatible through fetching by content address, but are out of scope for definition.
```

Comment:
definition is a narrow term..  I think best to just say spec.. or remove. Or did you mean to indicate that a non-definition discussion of some type may come out of the spec regarding said external methods?
/s/definition/this spec/

Reply:
I can remove it, what I was trying to say is that external methods will not be defined within this specification. Rather the interface external methods would need is already in place.